build_feathers.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# Build feather packages
+# Run with:
+#    docker run --rm -v $PWD:/io quay.io/pypa/manylinux1_x86_64 /io/build_feathers.sh
+# or something like:
+#    docker run --rm -e PYTHON_VERSIONS=2.7 -v $PWD:/io quay.io/pypa/manylinux1_x86_64 /io/build_feathers.sh
+# or:
+#    docker run --rm -e FEATHER_VERSIONS=0.17.0 -e PYTHON_VERSIONS=2.7 -v $PWD:/io quay.io/pypa/manylinux1_x86_64 /io/build_feathers.sh
+#
+# Make sure numpy and Cython wheels are on the manylinux server or built in the
+# $WHEELHOUSE directory first.
+set -e
+
+# Manylinux, Python versions
+if [ -z "${PYTHON_VERSIONS}" ]; then
+    PYTHON_VERSIONS="2.7 3.4 3.5"
+fi
```

Comment:
I am not sure about that. Should I instead skip PYTHON < 2.7 for Python 2 and PYTHON  < 3.4 for Python  3 in the main build loop?